cloud-on-k8s-nightly is failing

`cloud-on-k8s-nightly` is failing since January 1st:

```
02:05:55  Error: dirty local changes
02:05:55   M NOTICE.txt
02:05:55  Makefile:419: recipe for target 'check-local-changes' failed
02:05:55  make: *** [check-local-changes] Error 1
02:05:55  Makefile:36: recipe for target 'ci-internal' failed
```

--- a/NOTICE.txt
+++ b/NOTICE.txt
@@ -1,4 +1,4 @@
-Copyright 2018-2019 Elasticsearch BV
+Copyright 2018-2020 Elasticsearch BV
 
 This product includes software developed by The Apache Software
 Foundation (http://www.apache.org/).

